operators/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+    - knkski
+    - rfmvasconcelos
+
+reviewers:
```

Comment:
By fix the indentation, do you mean dedent the two list items? Those are like that to match the top-level `OWNERS` file, which also uses 4 spaces for indentation.

Reply:
@knkski Oh sorry, I just checked other OWNERS files in the various Kubeflow repos and they have 2 spaces instead of 4.
For example: https://github.com/kubeflow/internal-acls/blob/master/OWNERS.
I think we should fix it for the root OWNERS as well, but we can do it in a separate PR.